meta.yml

```
@@ -0,0 +1,120 @@
+---
+fullname: Zorn's Lemma
+shortname: zorns-lemma
+organization: coq-community
+
+synopsis: This library develops some basic set theory.
+
+description: |
+  This library develops some basic set theory.
+  The main purpose the author had in writing it was as support for the Topology library.
+
+authors:
+- name: Daniel Schepler
+  e-mail: dschepler@gmail.com
+  initial: true
+
+maintainers:
+- name: Andrew Miloradovsky
+  nickname: amiloradovsky
+
+opam-file-maintainer: miloradovsky@gmail.com
+
+license:
+  fullname: GNU Lesser General Public License v2.1 or later
+  identifier: LGPL-2.1-or-later
+  file: COPYING
+
+supported_coq_versions:
+  text: Coq 8.6 or later
+  opam: '{(>= "8.6" & < "8.10~") | (= "dev")}'
+
+tested_coq_versions:
+- version_or_url: https://github.com/coq/coq/tarball/master
```

It is now advised to use instead the URL https://github.com/coq/coq-on-cachix/tarball/master as this represents a (possibly slightly outdated) version of Coq master that has already been built and uploaded to Cachix, thus guaranteeing that the CI build will be fast:

```suggestion
- version_or_url: https://github.com/coq/coq-on-cachix/tarball/master
```